packages/composites/price/CHANGELOG.md

```diff
@@ -0,0 +1,40 @@
+# @chainlink/example-composite-adapter
```

Comment:
This should probably go to a more top-level directory.

Reply:
We've recently started versioning each EA package individually, so we'll need changelogs for each package